ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,145 @@
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  intermediate415:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.14"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-c2s-ipi-disc-priv-fips-f14
+  cron: 0 9 8,22 * *
+  steps:
+    cluster_profile: aws-c2s-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      UPGRADE_CHECK_RUN_TAGS: '@disconnected'
+      UPGRADE_PRE_RUN_TAGS: '@disconnected'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_TARGET}" >
+        ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE415
+        name: release:intermediate415
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: cucushift-upgrade-drop-last-hop
+    - ref: cucushift-chainupgrade-toimage
+    - chain: openshift-upgrade-qe-test-disconnected
+    workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-disc-priv-sts-basecap-none-fips-f14
+  cron: 46 3 12,28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      BASELINE_CAPABILITY_SET: None
+      EXPECTED_CAPABILITIES_IN_CREDENTIALREQUEST: MachineAPI
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+      PERSISTENT_MONITORING: "false"
+      TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      UPGRADE_CCO_MANUAL_MODE: oidc
+      UPGRADE_CHECK_RUN_TAGS: '@disconnected'
+      UPGRADE_PRE_RUN_TAGS: '@disconnected'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_TARGET}" >
+        ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE415
+        name: release:intermediate415
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: cucushift-upgrade-drop-last-hop
+    - ref: cucushift-chainupgrade-toimage
+    - chain: openshift-upgrade-qe-test-disconnected
```

Comment:
Just happen to find here is a disconnected profile, but the mirror image step seems not integrated into chainupgrade yet. Shall we update it to connected profile? or inlcude `cucushift-upgrade-mirror-images-by-oc-adm`  here.

Reply:
Agree, it is better to add a new chain for disconnected profile, something like `openshift-upgrade-qe-test-disconnected-chainupgrade`. BTW, this is the 1st time we are going to cover disconnected profile in chain upgrade, that is why we do not have that chain yet.

Reply:
@shellyyang1989 I remember you mentioned before we do not need to cover disconnected profile in chain upgrade, now the strategy changed?

Reply:
> mirroring images will consume more time and disk space in mirror registry

That's why I don't prefer to cover it. The only difference on upgrade between connected and disconnected clusters is ICSP, right? I think the feature can be covered in one-hop upgrade test.

Reply:
What's the decision?

Reply:
if all the other comments have been addressed, we can drop the disconnect ones and discuss it after holiday to not block the merging.